dbt_project.yml

```diff
@@ -199,6 +199,9 @@ models:
       base:
         +schema: nft_base
 
+    fungible:
+      +schema: fungible
+
```

Comment:
I don't get it, what do you mean?

Reply:
rather than assigning a schema in the project-wide file, just use:
`schema = 'fungible'`
in each model itself, the config block, by the alias

helps keep it clean and easy to follow of what the full table name will be. internally, we will do a universal cleanup at some point, it's just lower priority

Reply:
ohhh I thought adding to the `dbt_project.yml` file was necessary for all spells but that makes sense, just changed it!

Reply:
nope, not necessary, just another spot to add config settings. we want to work towards keeping it for more universal config settings, rather than model-specific configs which would make more sense in the model itself considering the scale of spellbook